frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/currentUserComment/currentUserComment.component.tsx

```diff
@@ -50,17 +56,47 @@ export const CurrentUserComment = ({
 	const [isEditMode, setIsEditMode] = useState(false);
 	const readOnly = TicketsCardHooksSelectors.selectReadOnly();
 
+	// @ts-ignore
+	const onDeleteImage = (index) => onEdit(_id, message, images.toSpliced(index, 1));
+
+	const onUploadImages = async () => {
+		const files = await uploadFile(getSupportedImageExtensions(), true) as File[];
+		const imagesToUpload = [];
+		for (const file of files) {
+			if (!imageIsTooBig(file)) {
+				imagesToUpload.push(await convertFileToImageSrc(file) as string);
+			}
+		}
+		onEdit(_id, message, images.concat(imagesToUpload));
+		const imagesTooBigCount = files.length - imagesToUpload.length;
+		if (imagesTooBigCount) {
+			DialogsActionsDispatchers.open('warning', {
+				title: formatMessage({
+					defaultMessage: 'Max file size exceeded',
+					id: 'comment.uploadImages.error.tooBig.title',
+				}, { imagesTooBigCount }),
+				message: formatMessage({
+					defaultMessage: `
+						{imagesTooBigCount} {imagesTooBigCount, plural, one {file was} other {files were}} too big and could not be uploaded.
+						The max file size is {maxFileSize}`,
+					id: 'comment.uploadImages.error.tooBig.message',
+				}, { imagesTooBigCount, maxFileSize: clientConfigService.resourceUploadSizeLimit }),
+			});
+		}
+	};
+	const imagesEditingFunctions = { onDeleteImage, onUploadImages };
```

Comment:
Since these are only being used in `<EditComment />` could they not be defined within that component instead?

Reply:
They are also used in the `Comment` within this file

Reply:
Ah yeah, I'm just blind